packages/core/.gitignore

```diff
@@ -0,0 +1 @@
+lib
```

Comment:
Can we put this into the top level `.gitignore`? `/packages/*/lib` should catch all?